rotkehlchen/db/history_events.py

```diff
@@ -684,6 +684,12 @@ def get_history_events_count(
         if entries_limit is None:
             return count_without_limit, count_without_limit
 
+        # When we have a limit but the total is already smaller or equal,
+        # just return the total for both counts
+        if count_without_limit <= entries_limit:
+            return count_without_limit, count_without_limit
+
```

Comment:
why this change? Could you point why this?

Reply:
we have a test, `test_get_events()`
https://github.com/rotki/rotki/blob/8e61ba1b8028ed4e00c8b61990edb417a9b45329/rotkehlchen/tests/api/test_history_base_entry.py#L569-L586

it handles the test case: `(2, True, 2, 2)` when the total count is less than or equal to the limit. In this case, we should just return the total count for both values rather than running an unnecessary limited query